Loenn/lang/en_gb.lang

```diff
@@ -132,3 +132,24 @@ entities.FactoryHelper/PistonRight.attributes.description.startActive=If `true`
 entities.FactoryHelper/PistonRight.attributes.description.heated=Whether the pipe is heated or not. A heated pipe kills the player when grabbed or stood on.
 entities.FactoryHelper/PistonRight.attributes.description.activationId=String value of the entity's Activation Id. When a factory activator with this Id is turned on the piston will toggle state.
 
+# Power Line
+entities.FactoryHelper/PowerLine.placements.name.power_line=Power Line
+entities.FactoryHelper/PowerLine.description.attributes.activationId=String value of the entity's Activation Id. When a factory activator with this Id is turned on the entity will toggle state.
```

Comment:
For all of these you flipped "description" and "attributes" so none of them appear